lib/functions/compilation/patch/kernel-bootsplash-and-drivers.sh

```diff
@@ -41,12 +21,12 @@ compilation_prepare()
 
 		chmod 755 ${kerneldir}/scripts/package/{builddeb,mkdebian}
 
-	elif linux-version compare "${version}" ge 5.8.17 \
-		&& linux-version compare "${version}" le 5.9 \
-		|| linux-version compare "${version}" ge 5.9.2; then
-			display_alert "Adjusting" "packaging" "info"
-			cd "$kerneldir" || exit
-			process_patch_file "${SRC}/patch/misc/general-packaging-5.8-9.y.patch" "applying"
+	elif linux-version compare "${version}" ge 5.8.17 &&
```

Comment:
possible linting problem as previously mentioned by @hzyitc

Reply:
This is the canonical, default, style of shellfmt... https://github.com/mvdan/sh/blob/master/syntax/canonical.sh